cmd/kyverno/Dockerfile

```diff
@@ -1,31 +1,8 @@
-# Multi-stage docker build
-# Build stage
-FROM golang:1.17.2 AS builder
+FROM gcr.io/distroless/static
```

Comment:
can we use `golang:1.17.3-alpine3.14`, which would have a very small footprint ?

Reply:
I believe `distroless` should have a significantly smaller footprint - from the distroless docs:
```
Distroless images are very small. The smallest distroless image, gcr.io/distroless/static-debian11, is around 2 MiB. That's about 50% of the size of alpine (~5 MiB), and less than 2% of the size of debian (124 MiB).
```
The specific image you mentioned is a build image - so it's a bit bigger even AFAIK. See https://hub.docker.com/layers/golang/library/golang/1.17.3-alpine3.13/images/sha256-1a52db9a19419e7e893dccd52a020cbbfaa7a0ea21f221a7be0d6206346b461a?context=explore